### ISSUE TITLE
Error with the subtract operator.

### ISSUE THREAD

**Issue by ortiga88** [influxdata/telegraf#8396]
When launching telegraf I debuted the following error:

```
> [inputs.sqlserver] Error in plugin: Script SQLServerProperties failed: mss
ql: The varchar and varchar data types are incompatible with the subtract operator.
```

any solution for this?

**Comment by sjwang90** [comment]
Can you provide your config you're using? If you would like to perform math operations like subtract, please check out the [Starlark processor](https://github.com/influxdata/telegraf/tree/master/plugins/processors/starlark).

We reserve Telegraf issues for bugs for reproducible problems.  Next time, please reach out to us in our [Community Slack](influxdata.com/slack) or [Community Page](https://community.influxdata.com/) and we can get this question answered quickly.

**Comment by ssoroka** [comment]
This is fixed in v1.16.2, which was just released and should be available now.
